tb_pulumi/cloudfront.py

```diff
@@ -78,14 +78,6 @@ def __init__(
         self.resources['logging_bucket'] = aws.s3.Bucket(
             f'{name}-loggingbucket',
             bucket=f'{service_bucket_name}-logs',
-            grants=[
-                {
-                    'permissions': ['FULL_CONTROL'],
-                    'type': 'CanonicalUser',
-                    'id': aws.s3.get_canonical_user_id().id,
-                    'uri': '',
-                }
-            ],
```

Comment:
AWS has disabled ACLs on S3 buckets by default. CloudFront needs this grant to work. Grants need ACLs enabled to work. Therefore, describing this grant here creates a circular dependency which cannot be resolved. For this reason, the BucketAclV2 resource is created below.